src/core/server/core_usage_stats/core_usage_stats_client.ts

```
@@ -0,0 +1,163 @@
+    const attributes = {
+      ...coreUsageStats,
+      apiCalls: {
+        ...apiCalls,
+        savedObjectsImport: {
+          total: current.total + 1,
+          createNewCopiesEnabled: incrementBooleanCounter(
+            current.createNewCopiesEnabled,
+            createNewCopies
+          ),
+          overwriteEnabled: incrementBooleanCounter(current.overwriteEnabled, overwrite),
+        },
+      },
+    };
+    await this.updateUsageStats(attributes);
+  }
+
+  public async incrementSavedObjectsResolveImportErrors({
+    createNewCopies,
+  }: IncrementSavedObjectsResolveImportErrorsOptions) {
+    const coreUsageStats = await this.getUsageStats();
+    const { apiCalls = {} } = coreUsageStats;
+    const {
+      savedObjectsResolveImportErrors: current = SAVED_OBJECTS_RESOLVE_IMPORT_ERRORS_DEFAULT,
+    } = apiCalls;
+
+    const attributes = {
+      ...coreUsageStats,
+      apiCalls: {
+        ...apiCalls,
+        savedObjectsResolveImportErrors: {
+          total: current.total + 1,
+          createNewCopiesEnabled: incrementBooleanCounter(
+            current.createNewCopiesEnabled,
+            createNewCopies
+          ),
+        },
+      },
+    };
+    await this.updateUsageStats(attributes);
+  }
+
+  public async incrementSavedObjectsExport({
+    types,
+    supportedTypes,
+  }: IncrementSavedObjectsExportOptions) {
+    const coreUsageStats = await this.getUsageStats();
+    const { apiCalls = {} } = coreUsageStats;
+    const { savedObjectsExport: current = SAVED_OBJECTS_EXPORT_DEFAULT } = apiCalls;
+    const isAllTypesSelected = !!types && supportedTypes.every((x) => types.includes(x));
+
+    const attributes = {
+      ...coreUsageStats,
+      apiCalls: {
+        ...apiCalls,
+        savedObjectsExport: {
+          total: current.total + 1,
+          allTypesSelected: incrementBooleanCounter(current.allTypesSelected, isAllTypesSelected),
+        },
+      },
+    };
+    await this.updateUsageStats(attributes);
```

we should swallow any exceptions from inside the `CoreUsageStatsClient` methods, we don't want these exceptions to bubble up to consuming plugins or for instance the import route causing a 500 because we couldn't persist usage data.

---

Great catch, I'll add it!